### ISSUE TITLE
enable rosenpass on android client is not possible

### ISSUE THREAD

**Issue by lfarkas** [netbirdio/netbird#1751]
if all on my netbird client I enable rosenpass then I can't communicate with android clients. since there is no possibility on android to enable it or edit somehow the config file. what's more there is no command line config for linux client to RosenpassPermissive to true.
so first of all it'd be useful to add a config flag on android (at least in expert mode) and write some kind of docs about it as well as the RosenpassPermissive settings...

**Comment by pascal-fischer** [comment]
Hi @lfarkas, that is correct, the mobile client do not yet allow to enable rosenpass. But there is an option to run rosenpass in permissive mode which would also allow mobile client to connect by setting `--rosenpass-permissive` during the up command. [https://docs.netbird.io/how-to/enable-post-quantum-cryptography#enable-rosenpass-in-net-bird](https://docs.netbird.io/how-to/enable-post-quantum-cryptography#enable-rosenpass-in-net-bird)

**Comment by lfarkas** [comment]
yes but it'd be nice to be at least an issue about the android support...

ps. and sometimes would be nice to update the android clients too:-)

**Comment by TechHutTV** [comment]
ndroid Rosenpass connectivity was fixed in PR #5044. Android peers can connect into a Rosenpass-enabled mesh, and `--rosenpass-permissive` on desktop peers covers mixed setups.
